sound/soc/sof/intel/hda.c

```diff
@@ -335,24 +335,30 @@ static int hda_init_caps(struct snd_sof_dev *sdev)
 	if (bus->ppcap)
 		dev_dbg(sdev->dev, "PP capability, will probe DSP later.\n");
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA)
+	/* init i915 and HDMI codecs */
+	ret = hda_codec_i915_init(sdev);
+	if (ret < 0) {
+		dev_err(sdev->dev, "error: no HDMI audio devices found\n");
```

Comment:
The error message here looks wired?
```suggestion
		dev_err(sdev->dev, "error: init i915 and HDMI codec failed\n");
```

Reply:
done!